New log file opened at Mon Aug 24 21:40:35 CEST 2015 by 6688@Obsidian
Taking next connection to: 53276
New client attempting to connect on port: 53277-53276
  Address = /192.168.1.144:53276
This client on port 53277 has uuid 662259fa-dfe0-44a2-b4f9-1002283f468f configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-662259fa-dfe0-44a2-b4f9-1002283f468f
Taking next connection to: 53276
New client attempting to connect on port: 53278-53276
  Address = /192.168.1.144:53276
Handshake exception on socket: 53278
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53276
New client attempting to connect on port: 53279-53276
  Address = /192.168.1.144:53276
This client on port 53279 has uuid e1eb9e41-3ee9-4ddf-9326-46070d827992 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-662259fa-dfe0-44a2-b4f9-1002283f468f, activator-e1eb9e41-3ee9-4ddf-9326-46070d827992
Taking next connection to: 53276
New client attempting to connect on port: 53292-53276
  Address = /192.168.1.144:53276
Handshake exception on socket: 53292
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53276
New client attempting to connect on port: 53293-53276
  Address = /192.168.1.144:53276
This client on port 53293 has uuid 8866421b-b064-42db-8e20-f781a95734b6 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-e1eb9e41-3ee9-4ddf-9326-46070d827992, play-fork-run-8866421b-b064-42db-8e20-f781a95734b6
Taking next connection to: 53276
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53276
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53276
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53276
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
Server socket thread exiting.
0 clients open, will close them...
All client sockets have been closed.
Closing the 6688@Obsidian logs at Mon Aug 24 21:53:12 CEST 2015, goodbye.
